src/core/utils/fileutils.cpp

```diff
@@ -58,9 +58,18 @@ bool FileUtils::copyRecursively( const QString &sourceFolder, const QString &des
   // Remove the destination folder and its content if it already exists
   if ( wipeDestFolder )
   {
+    qDebug() << "removing " + destFolder;
```

Comment:
Happy to leave these debug statements if they are more verbose to non involved readers

Reply:
@m-kuhn , I was thinking to at least leave one debug call along the lines of "Different git revision detected, copying data to share directory" so we can get some hint in logcat of users in the future.

Reply:
And a `%1 files copied to %2` at the end would also be nice

Reply:
... I'm thinking about moving this (at least partially) to all platforms. The package data will be r/o in *every* modern well maintained system (root only, admin only, in a :/assets file system ...) so we need to copy at least demo_projects to somewhere else.